.env.example

```diff
@@ -22,9 +22,4 @@ REDIS_URL=redis://localhost:6379
 # This is required by the authentication feature but can be left empty if you are disabling it when doing local tests
 
 ZITADEL_DOMAIN=https://auth.forms-staging.cdssandbox.xyz
-ZITADEL_APPLICATION_KEY=YOUR_APPLICATION_KEY
-
-# (optional) Zitadel but only used by the utilTokenGeneration script to generate access token
-
-PROJECT_ID=123 # This is the GCForms project ID in Zitadel
```

Comment:
This was for the `utilTokenGeneration.mjs` script that is being deleted with this PR